Library/Homebrew/formula.rb

```diff
@@ -308,9 +312,20 @@ def full_installed_alias_name
     full_name_with_optional_tap(installed_alias_name)
   end
 
+  def prefix_formula_file
```

Comment:
Not sure I understand the purpose of this method; is it to guard the file not existing? Not sure it's worth the `formula.rb` addition if so for a single caller.

Reply:
Yeah, that's fair. The goal was to remove the need for a string interpolation for this kind of thing when it's used elsewhere (which is uglier in my opinion) but it turns out that there aren't really any cases where a formula instance method can be used. Most of the time, the interpolation is needed because we are either using the file to create the formula object or doing simple operations where we don't need a full formula object.